string topic_name
string data_json
---
bool injected